tools/run_tests/artifact_targets.py

```diff
@@ -109,8 +109,8 @@ def build_jobspec(self):
       # TODO(atash) get better platform-detection support in core so we don't
       # need to do this manually...
       environ['CFLAGS'] = '-DGPR_MANYLINUX1=1'
-      environ['BUILD_HEALTH_CHECKING'] = 'TRUE'
-      environ['BUILD_MANYLINUX_WHEEL'] = 'TRUE'
+      environ['GRPC_BUILD_FIDDLY_BITS'] = 'TRUE'
```

Comment:
I think I'd like a name that is the least-general generalization of the uses of the flag. Obviously `*_FIDDLY_BITS` is insufficient, but it's used for more than docker. `GRPC_BUILD_EXTENSIONS`? `GRPC_BUILD_OPTIONALS`? `GRPC_BUILD_PROTO_DEPENDENTS`? I'unno. :expressionless: